### DISCUSSION TITLE
add Now indicator

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [kalkih/mini-graph-card#916] [Ideas]
for several configs, it would be awesome if we could have a Now indicator:

```
    now:
      show: true
      label: Now
```
is what apexchart card allows, and, because an image says more than a 1000 words:

<img width="471" alt="Scherm­afbeelding 2023-02-05 om 15 06 32" src="https://user-images.githubusercontent.com/33354141/216846177-2ad6412d-c124-4b37-9569-a3fc8c8292c8.png">

or the new [Plotly card](https://github.com/dbuezas/lovelace-plotly-graph-card#now-line)

**Comment by ildar170975** [comment]
In mini-graph-card the “now” point is the last one.
In the apexchart card the "now" feature is only useful to display "future" generated data (not historical).
The mini-graph-card does NOT support generated data, it only uses data from recorder.
